cli/ssync/main.go

```diff
@@ -5,13 +5,13 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/rancher/sparse-tools/log"
```

Comment:
In fact it will only take about half an hour to do so. I definitely prefer clean code at small expense.